babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+    '@babel/preset-react',
+  ],
+};
```

Comment:
- [ ] Please kindly ensure that babel is added as part of dependencies in your package.json file. I got the error message below when I tried to precompile your project's asset. 👇 


<img width="440" alt="Zion Women rally" src="https://github.com/kazim110/hello-rails-react/assets/103386183/2c80d390-627e-42cf-82b8-d7f9c9a913b3">